README.md

```diff
@@ -19,11 +19,31 @@ To blow away the created [enclave][enclaves-reference], run `kurtosis clean -a`.
 <details>
     <summary>Click to see configuration</summary>
 
-You can configure this package using a JSON structure as the provided arguments. 
-You can either provide the arguments manually 
+You can configure this package using a JSON structure as an argument to the `kurtosis run` function. The full structure that this package accepts are as follows, with default values shown (note that the `//` lines are not valid JSON and should be removed!):
+
+```javascript
+{
```

Comment:
I wanted to make sure users can see the structure, with default values, and explanations. (medium-term I believe we should have a productized way to do this)